plugins/Ntuplizer.cc

```diff
@@ -522,211 +553,247 @@ Ntuplizer::analyze(const edm::Event& iEvent, const edm::EventSetup& iSetup)
     //------------------------------------------------------------------------------------------
     //------------------------------------------------------------------------------------------
 
```

Comment:
Do this inside an ```if(isShort && isControl)``` statement, since Z & W candidates are not currently used (or even stored) for tracking lifetimes?

Reply:
Done!